modular_azurepeak/code/game/objects/items/donator_fluff_items.dm

```diff
@@ -55,6 +55,32 @@
 	icon = 'modular_azurepeak/icons/clothing/donor_clothes.dmi'
 	mob_overlay_icon = 'modular_azurepeak/icons/clothing/onmob/donor_clothes.dmi'
 
+//Bigfoot's donator item - knight helmet with gilded pattern
+/obj/item/clothing/head/roguetown/helmet/heavy/knight/gilded
+	name = "gilded knight's helmet"
+	desc = "A noble knight's helm made of steel and completed with a gilded trim."
+	icon_state = "gildedknight"
+	item_state = "gildedknight"
+	icon = 'modular_azurepeak/icons/clothing/donor_clothes.dmi'
+	mob_overlay_icon = 'modular_azurepeak/icons/clothing/onmob/donor_clothes.dmi'
+
+/obj/item/clothing/head/roguetown/helmet/heavy/knight/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/natural/feather) && !detail_tag)
+		user.visible_message(span_warning("[user] adds [W] to [src]."))
+		user.transferItemToLoc(W, src, FALSE, FALSE)
+		detail_tag = "_detail"
+		update_icon()
+		if(loc == user && ishuman(user))
+			var/mob/living/carbon/H = user
+			H.update_inv_head()
```

Comment:
this is missing a ..() I believe

Reply:
Purposefully missing to prevent it forcing color selection on the detail addition. Path is incorrect though, gotta fix that.